workflow/rules/qc.smk

```diff
@@ -16,7 +16,7 @@ rule multiqc:
     input:
         get_multiqc_input
     output:
-        "results/qc/multiqc/multiqc.html"
+        report("results/qc/multiqc/multiqc.html", caption="../report/multiqc_report.rst", category="QC")
```

Comment:
Really cool, that this simply works like this to generate a clickable link to the multiQC output in the report! To make sure this doesn't get lost in the deeptools fingerprint QC plots when there are more samples, I would give it its own report category, "multiQC", and rename "QC" to "other QC".